types/keys.d.ts

```diff
@@ -0,0 +1,2 @@
+export function keys<T extends object>(x: T): Array<keyof T>;
+export function keys<T>(x: T): string[];
```

Comment:
The generic parameter `T` in the second overload of `keys` is unnecessary, we can rewrite it to:

```ts
keys(o: object): string[];
```

which is a typescript official type.

Reply:
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#generics